### PR TITLE
Multiple Metrics

### PR DESCRIPTION
**Pull request by brandoncorey**
In this PR, the ability to add multiple queries to the dashboard is added. Fundamentally, the core state is more dynamic, with query being held in an object within an array, rather than hardcoded as single selections.

There is one area that may potentially be an issue, where I use an index as the key in a react component. It does not cause problems currently, but maybe can be replaced?

### CODE REVIEW

File: src/components/Dropdown.jsx
Comment (AlliLearns) [inline src/components/Dropdown.jsx:1]: Please add a clarifying comment about the possible value(s) of `items`.

File: src/components/FilterDropdownOptions.jsx
Comment (AlliLearns) [inline src/components/FilterDropdownOptions.jsx:1]: Suggested change: rename `value` to `attrValue` or `attributeValue`.

File: src/components/QueryBuilder.jsx
Comment (AlliLearns) [inline src/components/QueryBuilder.jsx:1]: Update `setQuery` to `setQueryState`

Also, use a UUID on lines 182 and 188 to identify the query.

File: src/components/ResultGraph.jsx
Comment (AlliLearns) [inline src/components/ResultGraph.jsx:1]: Grab first element of `queryData` array on line 94 and turn `queryData[0].aggregationType` into a variable.

**Review by AlliLearns — CHANGES_REQUESTED** [review]
Thanks! See comments for updates.

File: src/components/Dropdown.jsx
Comment (brandoncorey) [reply src/components/Dropdown.jsx:1]: changed to `availableOptions`

File: src/components/FilterDropdownOptions.jsx
Comment (brandoncorey) [reply src/components/FilterDropdownOptions.jsx:1]: changed to `attrValue`

File: src/components/ResultGraph.jsx
Comment (brandoncorey) [reply src/components/ResultGraph.jsx:1]: done

**Comment by brandoncorey** [comment]
The most substantial change is the UUID for the queries, I switched `queryState` to an object with the UUIDs as the key, but because before it was just an array being accessed with an index, the syntax didn't need to change for almost any of the code. Only when making the requests to the backend is there a slight change.

**Review by AlliLearns — COMMENTED** [review]
Great, looks good! Thanks!

**Pull request merged**